types/natural-compare-lite/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for natural-compare-lite 1.4
+// Project: https://github.com/litejs/natural-compare-lite
+// Definitions by: Doniyor Aliyev <https://github.com/doniyor2109>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function naturalCompare(a: string, b: string): number;
+
+declare global {
```

Comment:
I'd prefer if we just pretended it doesn't do this. It also doesn't do it unless `module.exports` fails